Formula/srtp.rb

```diff
@@ -18,7 +18,6 @@ class Srtp < Formula
 
   def install
     system "./configure", "--disable-debug", "--prefix=#{prefix}"
-    system "make", "test"
     system "make", "shared_library"
     system "make", "install" # Can't go in parallel of building the dylib
     libexec.install "test/rtpw"
```

Comment:
Need to remove this line if we're no longer doing `make test`; it looks like that build artifact no longer exists.

```suggestion
```

Reply:
My question is - do we need this artifact? I dropped `make test` due to the audit.

Reply:
It looks like we expect to use it in the test, so I'd say we probably actually should keep it (and keep `make test`).